plugin.rb

```diff
@@ -575,4 +575,32 @@ def self.enqueue_assign_hooks(event, payload)
       end
     end
   end
+
+  if defined?(DiscourseAutomation)
+    add_automation_scriptable('random_assign') do
+      field :assignees_group, component: :group
+      field :assigned_topic, component: :text
+
+      version 1
+
+      triggerables %i[point_in_time recurring]
+
+      script do |context, fields|
+        begin
+          next unless SiteSetting.assign_enabled
```

Comment:
If you do `next unless SiteSetting.try(:assign_enabled)` you can avoid the rescue of `NoMethodError`

Reply:
I already changed it before your review, but I think you didn't see it